mm.c

```diff
@@ -623,6 +648,40 @@ static void test_place_block_into_free_list()
     printf("Test passed.\n\n");
 }
 
+static void test_extend_heap() {
+	printf("Test extend_heap.\n");
+
+	char **test_heap_ptr;
+	char **initial_epilog_location;
+	char **initial_heap_end;
+	char **new_epilog_location;
+	size_t initial_heap_size = 8; // words
+	size_t new_block_size = 50; // words
+
+	// create initial heap and epilog taken space at the end
+	test_heap_ptr = mem_sbrk((initial_heap_size) * WORD_SIZE);
```

Comment:
It might be problematic to use men_sbrk here because it might interfere with our actual heap ? Does malloc work ?